plugins/frontend/backstage-plugin-argo-cd/src/components/ArgoCDHistoryCard.tsx

```diff
@@ -48,6 +49,47 @@ const HistoryTable = ({
   const supportsMultipleArgoInstances: boolean = Boolean(
     configApi.getOptionalConfigArray('argocd.appLocatorMethods')?.length,
   );
+  const protocol = (proto: string): string => {
```

Comment:
```suggestion
  const protocol = (protocol: string): string => {
```
I think we do not spare much with this shortening.